client/src/docs-ui/select-anchor/select-anchor.tsx

```diff
@@ -62,11 +57,50 @@ export class DocsSelectAnchor {
     }
   };
 
+  @Watch("page")
+  async computeOptionVNodes() {
+    const filterKey = this.page && getFilterKeyFromPage(this.page);
+    this.selectedOption = filterKey && this.selectedFilters?.[filterKey];
+    this.selectedOptionMetadata =
+      filterKey &&
+      this.selectedOption &&
+      (filterMetadataByOptionByName[filterKey][
+        this.selectedOption
+      ] as FilterOptionMetadata);
+
+    this.options =
+      this.sortedVersions &&
+      ((await Promise.all(
+        this.sortedVersions.map(async ([filterValue, filterRoute]) => {
+          if (filterValue !== this.selectedOption) {
+            const meta =
```

Comment:
Let's change this variable name to describe what the boolean represents

Reply:
Be more specific. What variable name?

Reply:
`meta`

Reply:
Renamed